Products/ZenEvents/events2/processing.py

```diff
@@ -442,8 +446,16 @@ def __call__(self, eventContext):
         # iterate over all event identifier plugins
         for name, plugin in evtIdentifierPlugins:
             try:
+                plugin_start_time = time.time()
                 eventContext.log.debug("running identifier plugin, name=%s, plugin=%s" % (name,plugin))
                 plugin.resolveIdentifiers(eventContext, self._manager)
+                duration = time.time() - plugin_start_time
+                if duration > self._manager.slowSegmentThreshold:
+                    if self._manager.LOG_PERF_AS_INFO:
+                        log.info("[pid %s] IdentifierPipe plugin %s took %.2f seconds" % (os.getpid(), plugin, duration))
+                    else:
+                        log.debug("[pid %s] IdentifierPipe plugin %s took %.2f seconds" % (os.getpid(), plugin, duration))
```

Comment:
In both of these calls to the `log` object, they should should just pass arguments instead of performing string interpolation (the `%` operator), .e.g.
`log.info("[pid %s] IdentifierPipe plugin %s took %.2f seconds", os.getpid(), plugin, duration)`